app/assets/javascripts/snowcrash/modules/mentions.js

```diff
@@ -0,0 +1,20 @@
+(function(window) {
+  window.Mentions = {
+    init: function(elements) {
+      tribute = new Tribute({
+        allowSpaces: function() { return false },
+        menuItemTemplate: function(item) {
+          return '<img src="' + item.original.avatar_url + '" width="24px" height="24px" > ' + item.string
+        },
+        noMatchTemplate: function() { return '' },
+        values: JSON.parse($('meta[name=mentionable-users]').attr('content'))
+      });
+
+      $('[data-behavior=mentions-scroll]').scroll(function(){
```

Comment:
```suggestion
      $('[data-behavior~=mentions-scroll]').scroll(function(){
```
This allows for multiple behavior stacking on a single element